### PR TITLE
fix: mock logo data in design drawer for custom logos, add key to colortheme radio selection

### LINKED ISSUE #4450: Create mock logo rather than fetching from aws in design drawer
Currently, we fetch the logo in the design drawer and then useEffect to update the default value in the attachment. This is unnecessary and we should mock the logo using an Object with custom data a la `transformImageFieldToEditForm`.

### PR DESCRIPTION
**Pull request by justynoh**
## Problem

Currently, we fetch the logo from s3 bucket in order to display it, but this creates some asynchrony issues in rendering the design drawer which increases the complexity of the render. Instead, we should just mock the logo data based on the information associated with the form.

Closes #4450 

## Solution
- Pull useEffect out into a container component to ensure that the useAdminForm query loads before we call useForm to render the drawer contents.
- Add keys to color theme radio to get the component to stop complaining in the console.

**Breaking Changes** 
- [X] No - this PR is backwards compatible

### CODE REVIEW

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        resetDesignStore: resetDesignStoreSelector(state),
+      }),
+      [],
+    ),
+  )
+
+  // Load existing start page and custom logo into drawer form
+  useEffect(() => {
+    if (!form) return
+    console.log('set')
```
Comment (timotheeg) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]: Let's drop console logging

```suggestion
```

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    setStartPageData({
+      ...form.startPage,
+      estTimeTaken: !form.startPage.estTimeTaken
+        ? ''
+        : form.startPage.estTimeTaken,
```
Comment (timotheeg) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]:
```suggestion
      estTimeTaken: form.startPage.estTimeTaken || ''
```

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                  type: 'image/jpeg',
+                }),
+                'size',
+                { value: form.startPage.logo.fileSizeInBytes },
+              ),
+              srcUrl: `${logoBucketUrl}/${form.startPage.logo.fileId}`,
+            },
+    })
+    if (form?.startPage.logo.state === FormLogoState.Custom)
+      setCustomLogoMeta(form?.startPage.logo)
```
Comment (timotheeg) [inline frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]: `form` is already tested for truthiness line 41, so the conditional access is not needed.

```suggestion
    if (form.startPage.logo.state === FormLogoState.Custom)
      setCustomLogoMeta(form.startPage.logo)
```

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx (hunk 1 above)
Comment (justynoh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]: oops, my bad!

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx (hunk 2 above)
Comment (justynoh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]: thanks!

File: frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx (hunk 3 above)
Comment (justynoh) [reply frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/DesignDrawer/DesignDrawerContainer.tsx]: you're right, thanks!

**Review by timotheeg — APPROVED** [review]

**Comment by timotheeg** [comment]
I resolved the conflict

**Pull request merged**